org.mythtv.MythTV.yaml

```diff
@@ -0,0 +1,1063 @@
+
+  - name: perl-File-Listing
+    sources:
+      - type: archive
+        url: https://cpan.metacpan.org/authors/id/P/PL/PLICEASE/File-Listing-6.16.tar.gz
+        sha256: 189b3a13fc0a1ba412b9d9ec5901e9e5e444cc746b9f0156d4399370d33655c6
+    buildsystem: simple
+    build-options:
+      env:
+        PERL5LIB: /app/lib/perl5
+        PERL_MM_OPT: INSTALL_BASE=/app
+        PERL_MB_OPT: --install_base /app
+    build-commands:
+      - perl Makefile.PL
+      - make && make install
+      - find /app/lib/perl5 -type f -exec chmod u+w {} \;
+
+  - name: perl-IO-HTML
+    sources:
+      - type: archive
+        url: https://cpan.metacpan.org/authors/id/C/CJ/CJM/IO-HTML-1.004.tar.gz
+        sha256: c87b2df59463bbf2c39596773dfb5c03bde0f7e1051af339f963f58c1cbd8bf5
+    buildsystem: simple
+    build-options:
+      env:
+        PERL5LIB: /app/lib/perl5
+        PERL_MM_OPT: INSTALL_BASE=/app
+        PERL_MB_OPT: --install_base /app
+    build-commands:
+      - perl Makefile.PL
+      - make && make install
+      - find /app/lib/perl5 -type f -exec chmod u+w {} \;
+
+  - name: perl-LWP-MediaTypes
+    sources:
+      - type: archive
+        url: https://cpan.metacpan.org/authors/id/O/OA/OALDERS/LWP-MediaTypes-6.04.tar.gz
+        sha256: 8f1bca12dab16a1c2a7c03a49c5e58cce41a6fec9519f0aadfba8dad997919d9
+    buildsystem: simple
+    build-options:
+      env:
+        PERL5LIB: /app/lib/perl5
+        PERL_MM_OPT: INSTALL_BASE=/app
+        PERL_MB_OPT: --install_base /app
+    build-commands:
+      - perl Makefile.PL
+      - make && make install
+      - find /app/lib/perl5 -type f -exec chmod u+w {} \;
+
+  - name: perl-MIME-Base32
+    sources:
+      - type: archive
+        url: https://cpan.metacpan.org/authors/id/R/RE/REHSACK/MIME-Base32-1.303.tar.gz
+        sha256: ab21fa99130e33a0aff6cdb596f647e5e565d207d634ba2ef06bdbef50424e99
+    buildsystem: simple
+    build-options:
+      env:
+        PERL5LIB: /app/lib/perl5
+        PERL_MM_OPT: INSTALL_BASE=/app
+        PERL_MB_OPT: --install_base /app
+    build-commands:
+      - perl Makefile.PL
+      - make && make install
+      - find /app/lib/perl5 -type f -exec chmod u+w {} \;      
+
+  - name: perl-URI
+    sources:
+      - type: archive
+        url: https://cpan.metacpan.org/authors/id/O/OA/OALDERS/URI-5.31.tar.gz
+        sha256: b9c4d58b2614b8611ae03a95a6d60ed996f4b311ef3cd5a937b92f1825ecc564
+    buildsystem: simple
+    build-options:
+      env:
+        PERL5LIB: /app/lib/perl5
+        PERL_MM_OPT: INSTALL_BASE=/app
+        PERL_MB_OPT: --install_base /app
+    build-commands:
+      - perl Makefile.PL
+      - make && make install
+      - find /app/lib/perl5 -type f -exec chmod u+w {} \;
+
+  - name: perl-HTTP-Message
+    sources:
+      - type: archive
+        url: https://cpan.metacpan.org/authors/id/O/OA/OALDERS/HTTP-Message-7.00.tar.gz
+        sha256: 5afa95eb6ed1c632e81656201a2738e2c1bc6cbfae2f6d82728e2bb0b519c1dc
```

Comment:
but the perl sources in a seperate yaml and use the same build command for it. See https://github.com/flathub/com.gitlab.davem.ClamTk/blob/cf30cf7d7d0ec83e7e7fb88518caca135a857188/com.gitlab.davem.ClamTk.yaml#L62-L72 for inspiration

Reply:
> but the perl sources in a seperate yaml and use the same build command for it. See https://github.com/flathub/com.gitlab.davem.ClamTk/blob/cf30cf7d7d0ec83e7e7fb88518caca135a857188/com.gitlab.davem.ClamTk.yaml#L62-L72 for inspiration

Not sure where that packages install.sh is coming from.  Some of these perl packages have different build processes, some use Makefile.PL, some call it Build.PL.  I wouldn't be able to cleanly iterate through them

Reply:
Right now this seems unmaintainable

Reply:
>  I wouldn't be able to cleanly iterate through them

Didn't you use flatpak-builder-tools ? https://github.com/flatpak/flatpak-builder-tools/tree/master/cpan

If yes then these auto generated manifest are kept in a seperate files.
If no, then you should.